packages/flutter/lib/src/material/snack_bar_theme.dart

```diff
@@ -61,6 +61,8 @@ class SnackBarThemeData with Diagnosticable {
     this.shape,
     this.behavior,
     this.width,
+    this.icon,
+    this.insetPadding,
```

Comment:
the biggest nit of all time, feel free to ack, but I'd switch the order so that `insetPadding` is next to `width`

Reply:
Ha, no problem. Makes sense to group them together!